### PR TITLE
Feature/WD-18 Features Pages

### PR DESCRIPTION
**Pull request by l-alexandrov**
## Description

Added features pages:
* Upvote
* Roadmap
* FAQ
* Showcase of the latter 

Fixes [WD-18](https://bimbala.atlassian.net/browse/WD-18)

## Type of change

Please delete options that are not relevant.

- [X] New feature (non-breaking change which adds functionality)

## Is the solution different from the one proposed during the grooming?

No.

## How Has This Been Tested?

- [X] Locally in browser

# Checklist

Please delete the options that are not relevant.

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] My changes generate no new warnings

[WD-18]: https://bimbala.atlassian.net/browse/WD-18?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

## Summary by CodeRabbit

- **New Features**
	- Introduced a dedicated Features page, including popular features with images, titles, and links.
	- Added an FAQ page for commonly asked questions.
	- Launched a Roadmap page to showcase upcoming developments.
	- Implemented an Upvote page to engage users in feedback and feature voting.

- **Enhancements**
	- Enhanced the footer with a new "Features" section for improved navigation.

- **Documentation**
	- Updated web routes to include new feature-related pages.

### CODE REVIEW

**Comment by MrGKanev** [comment]
I tried fixing it, but it broke me internally. The problem I think is regarding the gradient or at least this is the most similar issue I found on [StackOverflow](https://stackoverflow.com/questions/54949659/white-line-appears-at-end-of-gradient-filled-div-at-specific-browser-widths).

P.S. I'm not leaving without an alternative. - A [simpler solution](https://play.tailwindcss.com/Gfvw15la7g), it isn't any similar to what you did, but it shows off one major feature and 2 less important ones. You can use something If it helps.

**Comment by l-alexandrov** [comment]
I'll check them out. Thanks!

File: resources/views/themes/bimbala/features/roadmap.blade.php [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        <ul class="list-disc list-inside leading-loose">
+            <li>Be transparent about your company's workflow and what the process is. This will prove your quality of work and will help you build brand identity</li>
+            <li>Easy integration of Bimbala with other task management tools</li>
+            <li>Encrouge your users to give their feedback by making their requests to work-in-progress tasks</li>
+        </ul>
+    </div>
+    <div class="col-span-6 text-center pr-24 grid justify-items-end">
+        <img src="{{ asset('themes/bimbala/images/features/workflow.svg') }}" class="max-h-[28rem]"/>
+    </div>
+</section>
```
[the remark this answers is not in the document]
Comment (l-alexandrov) [reply resources/views/themes/bimbala/features/roadmap.blade.php:51]: Yes.

**Comment by l-alexandrov** [comment]
@coderabbitai review

**Review by MrGKanev — APPROVED** [review]

**Pull request merged**